index.html

```diff
@@ -0,0 +1,221 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/695f3209b8.js" crossorigin="anonymous"></script>
+    <title>Portfolio Desktop</title>
+</head>
+
+<body>
```

Comment:
The Mobile section should not be changed or broken. Please make sure all the sections of the mobile page looks the same as in the design. Kindly see some screenshots below of the broken mobile page and make sure the mobile page is as it's before.
![mob-headline](https://user-images.githubusercontent.com/35504149/135472074-d63c9aea-c97b-4192-afee-b45b7c51edbd.png)
![mob-form](https://user-images.githubusercontent.com/35504149/135472110-620bbcee-0901-4f96-946e-1fd51a2e949f.png)

![mob-nav](https://user-images.githubusercontent.com/35504149/135472149-d86d13e6-6a22-487f-99eb-73c8df639c9c.png)